response_operations_ui/templates/reset-password.html

```diff
@@ -4,129 +4,124 @@
 {% from "components/password/_macro.njk" import onsPassword %}
 {% from "components/input/_macro.njk" import onsInput %}
 {% from "components/fieldset/_macro.njk" import onsFieldset %}
+{% from "components/table/_macro.njk" import onsTable %}
+{% from "components/checkboxes/_checkbox-macro.njk" import onsCheckbox %}
+{% from "components/panel/_macro.njk" import onsPanel %}
 
-{% set errorType = data['error']['type'] %}
-
-{% set page_title = "Reset Password" %}
+{% set page_title = 'Reset Password' %}
 
 {% block main %}
-
-    {% if errorType %}
-
-        {% set errorNotEnteredTitle = "Password is required" %}
-        {% set errorNotEnteredDescription = "Please check the passwords and try again" %}
-
-        {% set errorNoMatchTitle = "Your passwords do not match" %}
-        {% set errorNoMatchDescription = "Please check the passwords and try again" %}
-
-        {% set errorStrongerPasswordTitle = "Your password doesn't meet the requirements" %}
-        {% set errorStrongerPasswordDescription = "Please choose a different password" %}
-
-        {% set errorPasswordSameAsOldTitle = "Your password matches your old password" %}
-        {% set errorPasswordSameAsOldDescription = "Please choose a different password or login with the old password" %}
-
-        {% if errorType.password[0] == errorNotEnteredTitle %}
-            {% set errorTitle = errorNotEnteredTitle %}
-            {% set errorDescription = errorNotEnteredDescription %}
-        {% elif errorType.password[0] == errorNoMatchTitle %}
-            {% set errorTitle = errorNoMatchTitle %}
-            {% set errorDescription = errorNoMatchDescription %}
-        {% elif errorType.password[0] == errorStrongerPasswordTitle %}
-            {% set errorTitle = errorStrongerPasswordTitle %}
-            {% set errorDescription = errorStrongerPasswordDescription %}
-        {% elif errorType.password[0] == errorPasswordSameAsOldTitle %}
-            {% set errorTitle = errorPasswordSameAsOldTitle %}
-            {% set errorDescription = errorPasswordSameAsOldDescription %}
+    <div class="ons-grid">
+        <div class="ons-grid__col ons-col-8@m">
+        {% with messages = get_flashed_messages(category_filter=["error"]) %}
+            {% if messages %}
+                {% call
+                    onsPanel({
+                            "type": "error",
+                            "classes": "ons-u-mb-l",
+                            "title":  errorTitle
+                        })
+                %}
+                    {% for message in messages %}
+                        <p id="flashed-message-{{ loop.index }}">{{ message }}</p>
+                    {% endfor %}
+                {% endcall %}
+            {% endif %}
+        {% endwith %}
+          {% if form.password.errors or form.password_confirm.errors %}
```

Comment:
Is there anyway this can be written to a separate html page so it is not repeated for both and therefore reusable or is too different and I am not seeing that

Reply:
I've extracted it out into it's own partial html